hack/update-attributions.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+set -eoux pipefail
+
+SPLIT="----------------------------------------------------------------------"
+
+cat <<EOF > ATTRIBUTIONS.md
+ATTRIBUTIONS
```

Comment:
```suggestion
# Attributions
```

maybe, to at least have a bit of Markdown styling.

Reply:
I changed the header to be `h1` and each component to be `h2`. Thank you, it does look better. :+1: